examples/mobile/UIComponents/components/buttons/contained/index.html

```diff
@@ -11,6 +11,9 @@
 
 <body>
 	<div class="ui-page" id="ContainedButtonsDemo">
+		<div class="ui-appbar-left-icons-container">
+			<a href="#" class="ui-btn ui-btn-icon ui-btn-icon-back" data-style="flat" data-rel="back"></a>
+		</div>
```

Comment:
please double check code here, view is broken in this example
![image](https://user-images.githubusercontent.com/2246331/87505524-5baab000-c669-11ea-8204-14c9702241a1.png)